README.md

```diff
@@ -27,255 +20,169 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
-
-Run the following commands to launch Deephaven for Groovy / Java server applications. 
+TL;DR for Groovy:
 
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/groovy/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
+```sh
+git clone https://github.com/deephaven/deephaven-core.git
+cd deephaven-core
+./gradlew server-jetty-app:run -Pgroovy
 ```
 
-### Launch: Groovy / Java with example data
+### Required dependencies
 
-Run the following commands to launch Deephaven for Groovy / Java server applications, with example data.
-
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/groovy-examples/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
+Building and running Deephaven requires a few software packages.
 
-### Monitor logs
-
-The `-d` option to `docker-compose` causes the containers to run in the background, in detached mode.  This option allows you to use your shell after Docker launches the containers.
+| Package        | Version                       | OS           |
+| -------------- | ----------------------------- | ------------ |
+| git            | ^2.25.0                       | All          |
+| java           | >=11, <20                     | All          |
+| docker         | ^20.10.8                      | All          |
+| docker-compose | ^1.29.0                       | All          |
+| Windows        | 10 (OS build 20262 or higher) | Only Windows |
+| WSL            | 2                             | Only Windows |
 
-Since the container is running detached, you will not see any logs. However, you can follow the logs by running:
+You can check if these packages are installed and functioning by running:
 
 ```bash
-docker-compose logs -f
+git version
+java -version
+docker version
+docker-compose version
+docker run hello-world
 ```
 
-Use CTRL+C to stop monitoring the logs and return to a prompt.
-
-### Shutdown
-
-The deployment can be brought down by running:
-
-```bash
-docker-compose down
-```
+:::note
 
-### Manage example data
+Internally, the Java build process will use [Gradle Auto Provisioning](https://docs.gradle.org/current/userguide/toolchains.html#sec:provisioning)
+to download and use the appropriate Java version for building and testing.
 
-[Deephaven's examples repository](https://github.com/deephaven/examples) contains data sets that are useful when learning 
-to use Deephaven. These data sets are used extensively in Deephaven's documentation and are needed to run some examples. [Deephaven's examples repository](https://github.com/deephaven/examples) contains documentation on the available data sets and how to manage them. 
+On Windows, all commands must be run inside a WSL 2 terminal.
 
-If you have chosen a deployment with example data, the example data sets will be downloaded. Production deployments containing your own data will not need the example data sets.
+:::
 
+### Authentication
```

Comment:
This section needs to be more verbose/hand-holdy.  I think it will be a big stumbling block for new users.  Can the lowest common denominator make it through these docs?